plugins/common/common.go

```diff
@@ -343,3 +348,14 @@ func VerifyImage(image string) bool {
 	imageCmd.ShowOutput = false
 	return imageCmd.Execute()
 }
+
+//PlugnTrigger fire the given plugn trigger with the given args
+func PlugnTrigger(triggerName string, args ...string) (string, error) {
+	shellArgs := make([]interface{}, 0, len(args))
```

Comment:
i have some questions about this function
- why is `shellArgs` a `[]interface{}` and not `[]string`?
- why are we setting the len to 0 and cap to length and not just setting the len to `len(args) + 2` and using the index from range to set shellArgs?

Reply:
1) sh.Command is declared as `Command(name string, a ...interface{})` and I couldn't find a good way to convert `[]string{}` to `[]interface{}` and didn't want to leak `[]interface{}` into callers. Any advice?

2)I figured append may be more readable and less likely to mess up indexing if edited down the line, but reading it again yeah switching to indexing is a better call.

Reply:
@josegonzalez Is it necessary that plugn goes through `sh.Command` and not the `os/exec`package? If it's moved to use `os/exec` it will get simpler because the `NewShellCmd` stuff can get used. Does it rely on actually being executed in a shell?